.github/tests/javascript/graphql-sdl-first/run.sh

```diff
@@ -2,13 +2,14 @@
 
 set -eu
 
-yarn 
-yarn prisma db push --preview-feature
-yarn dev &
+npm install
+npx prisma migrate dev --name init
+npx prisma db seed --preview-feature
+npm run dev &
```

Comment:
Hmm, no yarn lockfile in the repo that also needs upgrading? No other uses of `yarn` in script or similar? Then this might be fine although in Engineering we usually use yarn.

Reply:
`npm` is used to mirror the commands the user/learner would use when following the instructions on the README file of the example. `yarn` would work just fine as well.

Reply:
Checked myself, we are not using lockfiles in this repo so this seems fine.